app/views/admin_posts/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="news admin home">
-  <% if logged_in_as_admin? %>
+  <% if policy(@admin_posts).can_post? %>
```

Comment:
It is. `policy` uses the argument to figure out which policy class to use (Pundit [can auto-detect classes from relation objects](https://github.com/varvet/pundit/blob/df96d2ae6bcf28991c1501d5ff0bde4c42aa4acd/lib/pundit/policy_finder.rb#L62-L63)) then send the argument to the policy class (AdminPostPolicy, which doesn't care if it gets an object or a collection of objects because we set it up that way).